datahub-web-react/src/app/shared/tags/Access/accessTerm.tsx

```diff
@@ -0,0 +1,28 @@
+import { BookOutlined } from '@ant-design/icons';
+import { Tag } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import Highlight from 'react-highlighter';
+
+const highlightMatchStyle = { background: '#ffe58f', padding: '0' };
+
+const StyledTag = styled(Tag)<{ fontSize?: number }>`
+    ${(props) => props.fontSize && `font-size: ${props.fontSize}px;`}
+`;
+
+interface Props {
+    data?: string;
```

Comment:
What is this data? Let's name it something like

RoleName? or RolePermission

isn't that what is shown using this component?

Reply:
removed the page as we cannot make changes in the default preview card